lib/kafka_rest/consumer.rb

```diff
@@ -0,0 +1,41 @@
+module KafkaRest
+  class Consumer
+    def self.create(client, group, id, format, opts={})
+      consumer = client.request(:post, "/consumers/#{group}", {
+        id: id,
+        format: format,
+        'auto.offset.reset' => opts['auto.offset.reset'] || 'smallest',
+        'auto.commit.enable' => opts['auto.commit.enable'] || 'true'
+      })
+      self.new(consumer[:instance_id], consumer[:base_uri], format)
+    end
+
+    attr_reader :group, :instance_id, :base_uri, :format
+
+    def initialize(group, instance_id, base_uri, format)
+      @group = group
+      @instance_id = instance_id
+      @base_uri = base_uri
+      @format = format
+    end
+
+    def delete
```

Comment:
Not sure if `delete` is the right name. This deregisters an instance, but doesn't delete the entire consumer(group).

Should we have a separate class for ConsumerGroup and ConsumerInstance, to avoid these kinds of confusions? Or just rename this to `deregister` or something like that?